mmdet/apis/inference.py

```diff
@@ -164,7 +163,8 @@ def show_result_pyplot(model,
                        score_thr=0.3,
                        fig_size=(15, 10),
                        title='result',
-                       block=True):
+                       block=True,
```

Comment:
The argument block does not actually work because it is not passed to `model.show_result`.